hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneAcl.java

```diff
@@ -170,7 +172,7 @@ public static OzoneAcl parseAcl(String acl)
 
     // TODO : Support sanitation of these user names by calling into
     // userAuth Interface.
-    return new OzoneAcl(aclType, parts[1], acls, aclScope);
+    return new OzoneAcl(aclType, parts[1], aclScope, validateAndCopy(acls));
```

Comment:
Here `validateAndCopy` can be ommitted, since the `BitSet` is created in `OzoneAcl`.  Better yet, construct an `EnumSet`.